pkg/reconciler/v1alpha1/pipelinerun/pipelinerun.go

```diff
@@ -186,21 +186,40 @@ func (c *Reconciler) reconcile(ctx context.Context, pr *v1alpha1.PipelineRun) er
 			})
 			return nil
 		}
-		return fmt.Errorf("error getting Tasks and/or TaskRuns for Pipeline %s: %s", p.Name, err)
+		return fmt.Errorf("error getting Tasks for Pipeline %s: %s", p.Name, err)
 	}
-	prtr := resources.GetNextTask(pr.Name, pipelineState, c.Logger)
+	for _, rprt := range pipelineState {
+		err := taskrun.ValidateResolvedTaskResources(rprt.PipelineTask.Params, rprt.ResolvedTaskResources)
+		if err != nil {
+			c.Logger.Error("Failed to validate pipelinerun %s with error %v", pr.Name, err)
+			pr.Status.SetCondition(&duckv1alpha1.Condition{
+				Type:    duckv1alpha1.ConditionSucceeded,
+				Status:  corev1.ConditionFalse,
+				Reason:  ReasonFailedValidation,
+				Message: err.Error(),
+			})
+			return nil
+		}
+	}
+	err = resources.ResolveTaskRuns(c.taskRunLister.TaskRuns(pr.Namespace).Get, pipelineState)
+	if err != nil {
+		return fmt.Errorf("error getting TaskRunss for Pipeline %s: %s", p.Name, err)
```

Comment:
typo "TaskRunss"

Reply:
whoops thanks :sweat_smile:

(it's `PipelineParamses` all over again XD)